README.md

```diff
@@ -56,12 +57,12 @@ gcloud ml-engine local train --package-path trainer \
 ## Deploy your trained model
 To deploy your model to ML Engine
 ```
-gcloud ml-engine models create MODEL_NAME
+gcloud ml-engine models create MODEL_NAME --regions=REGION
 gcloud ml-engine versions create VERSION --model=MODEL_NAME --origin=ORIGIN
 ```
 To test the deployed model:
 ```
-python predictions/predict.py
```

Comment:
This one should stay, this is an example of how to get predictions using python instead of the gcloud ml-engine predict command